src/quickfix.c

```diff
@@ -1186,10 +1205,11 @@ qf_init_ext(
 	convert_setup(&state.vc, enc, p_enc);
 #endif
     fields.namebuf = alloc_id(CMDBUFFSIZE + 1, aid_qf_namebuf);
+    fields.module = alloc_id(CMDBUFFSIZE + 1, aid_qf_namebuf);
```

Comment:
The aid_qf_namebuf identifier cannot be used for this. You need to create
a new alloc identifier for module name (aid_qf_module). A new enum should
be added to alloc.h.